### DISCUSSION TITLE
Recommendations not shown on screen

### DISCUSSION THREAD

**Discussion by ghost** [monarc-project/MonarcAppFO#466] [Q&A]
Hello, we are using Monarc v 2.12.2 on local installation.
Everything went fine but then suddenly the recommendations will not be shown at the risk sheets. This happens to booth the newly created recommendation as well as those imported from the knowledge base. Consequently it is not possibly to delete a recommendation. There is clear indication that the recommendations has been linked inside the database. So e.g. an error will be thrown if we try to reconnect the same recommendation to the risk, stating that this has already done. Another indication is, that the recommendations will appear in the reports.
We already delete the cache. What else should we do? How can we at least delete recommendations while working on an analysis?

Thanks
UJ

**Comment by ghost** [comment]
Done!

Well, no idea how it happen. But inspecting the issue once again the recommendations are back to the risk sheets.

**Comment by ruslanbaidan** [comment]
Hello,

Recommendations removal from the Knowledge Base works well on our cloud v2.12.3-p3.
It is possible to remove recommendations that have already been lined to a risk.
It is also possible to remove recommendations that are imported or created manually.

In case if you will face with the issue again, try to update the Monarc version to the latest.

Regards
